testing/test_pathlib.py

```diff
@@ -259,20 +277,6 @@ def test_check_filepath_consistency(
         assert orig == p
         assert issubclass(ImportPathMismatchError, ImportError)
 
-    def test_issue131_on__init__(self, tmp_path: Path) -> None:
```

Comment:
This test is actually incorrect, it was considering `proja` and `sub/proja` to be the same module, but this is clearly incorrect because there is no namespace package setup here.

We now have a proper test for namespace packages, so decided to just drop the faulty test instead of fixing it.

Reply:
Yes I'm not clear on why the tests thought this was desirable behavior or what the comment is trying to say (maybe about "pkgutil" namespace package? Hopefully we can not care about those). The original issue seems to be lost. So I think it's OK to drop this test.